projects/common-lib/src/lib/cards/cards.component.html

```diff
@@ -1,19 +1,19 @@
 <section id="canya-cards">
   <div class="container">
     <div class="row">
-      <div class="col-4" *ngFor="let app of apps">
+      <div class="col-lg-4 col-md-6 col-sm-12" *ngFor="let app of apps">
         <a class="card" [attr.href]="app.url" target="_blank" rel="noopener">
           <div class="img-wrapper">
-            <img [attr.src]="app.img" [attr.alt]="app.name">
+            <img style="max-width:75%; padding-top:15px;" [attr.src]="app.img" [attr.alt]="app.name">
```

Comment:
Let's add these styles to the `cards.component.scss` file to avoid having inline styles everywhere.